dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -56,7 +54,7 @@ public RetryingChannel(Channel delegate) {
 
         Supplier<ListenableFuture<Response>> callSupplier = () -> delegate.execute(endpoint, request);
         FutureCallback<Response> retryer = new RetryingCallback<>(callSupplier, future);
-        Futures.addCallback(callSupplier.get(), retryer, DIRECT_EXECUTOR);
+        Futures.addCallback(callSupplier.get(), retryer, MoreExecutors.directExecutor());
```

Comment:
did you intend to also refactor these to call MoreFutures.addDirectCallback?

Reply:
I refactored and then got all the warnings about ignoring a returned Future so flipped em back. The main benefit of the MoreFutures method is the fact that it returns the future which isn't necessary here. switched these back to have a constant for the executor